dll/win32/kernel32/client/proc.c

```diff
@@ -4181,7 +4181,7 @@ CreateProcessInternalW(IN HANDLE hUserToken,
 
     /* Create the Thread's Context */
     BaseInitializeContext(&Context,
-                          Peb,
+                          RemotePeb,
```

Comment:
While here, it looks like [line 3558](https://github.com/reactos/reactos/blob/bdf7e0a990722cee28b106426fc82970c5e0848f/dll/win32/kernel32/client/proc.c#L3558) could/should use `Peb`, for trivial consistency/optimization.